### DISCUSSION TITLE
How to export tables with alias-fieldnames

### DISCUSSION THREAD

**Discussion by urskaufmann** [QGEP/QGEP#705] [Q&A] [answered]
How can we export qgep-data to e.g. csv-format, so that the recipient has the same values and fieldnames as it is in the QGEP-project. The values (not codes, but text) is no problem. But with the fieldnames (they are English in the views or tables and are localized with alias)? Is it possible to export table-data with alias? Thanks for hints...

**Comment by sjib** [comment]
No - not possible ->Feature Request for this handed in as QGIS-CH Sponsoring Project Proposal 2021

**Comment by urskaufmann** [comment, marked as answer]
Was choosen as Sponsoring Project! Projekt 8: Benutzerdefinierte Feldnamen beim Export in QGIS
[Candidates for QGIS-CH sponsoring projects 2022.pdf](https://github.com/QGEP/QGEP/files/8960695/Candidates.for.QGIS-CH.sponsoring.projects.2022.pdf)

Selected candidates in the protocol:
https://www.qgis.ch/de/verein/generalversammlung/2022/protokoll-der-gv-2022
